README.md

```diff
@@ -112,7 +112,6 @@ We're back! Use this repo to share and keep track of software, tech, CS, PM, qua
 | **[Caldera](https://simplify.jobs/c/Caldera)** | Software Engineering Intern | SF | <a href="https://jobs.ashbyhq.com/Caldera/55c83409-bde2-4edb-97c5-4b57a5a74ac6/application?utm_source=Simplify&ref=Simplify"><img src="https://i.imgur.com/w6lyvuC.png" width="84" alt="Apply"></a> <a href="https://simplify.jobs/p/f21a4379-7670-4d6c-bc57-6cb7cd471d28?utm_source=GHList"><img src="https://i.imgur.com/aVnQdox.png" width="30" alt="Simplify"></a> | Feb 20 |
 | **[Bose](https://simplify.jobs/c/bose)** | Embedded Software Engineering Intern / Co-op | Framingham, MA | <a href="https://boseallaboutme.wd1.myworkdayjobs.com/en-US/Bose_Careers/job/US-MA---Framingham/Embedded-Software-Engineering-Intern---Co-op_R26404?utm_source=Simplify&ref=Simplify"><img src="https://i.imgur.com/w6lyvuC.png" width="84" alt="Apply"></a> <a href="https://simplify.jobs/p/3af3c2fb-47bb-4fe2-8df3-db62d0ef1483?utm_source=GHList"><img src="https://i.imgur.com/aVnQdox.png" width="30" alt="Simplify"></a> | Feb 20 |
 | **[Astranis](https://simplify.jobs/c/Astranis)** | Production Development – Intern - Summer 2024 | SF | <a href="https://boards.greenhouse.io/astranis/jobs/4337902006?utm_source=Simplify&ref=Simplify"><img src="https://i.imgur.com/w6lyvuC.png" width="84" alt="Apply"></a> <a href="https://simplify.jobs/p/5fcbbda7-a823-4b48-aa53-a4a0dda5687a?utm_source=GHList"><img src="https://i.imgur.com/aVnQdox.png" width="30" alt="Simplify"></a> | Feb 20 |
-| **[Xylem](https://simplify.jobs/c/Xylem)** | Software Engineering Intern | Morrisville, NC | <a href="https://jobs.jobvite.com/xylem/job/oX8CrfwS?nl=1&nl=1&fr=false&utm_source=Simplify&ref=Simplify"><img src="https://i.imgur.com/w6lyvuC.png" width="84" alt="Apply"></a> <a href="https://simplify.jobs/p/f1fb91ee-13e3-4f9d-8480-2e0142f770b7?utm_source=GHList"><img src="https://i.imgur.com/aVnQdox.png" width="30" alt="Simplify"></a> | Feb 19 |
```

Comment:
Do not edit the README directly.

https://github.com/SimplifyJobs/Summer2024-Internships/blob/dev/CONTRIBUTING.md#editing-an-internship